src/Copy/Text.elm

```diff
@@ -358,3 +358,13 @@ chompTrailingUrlSlash urlString =
 urlToDisplay : String -> String
 urlToDisplay url =
     Url.fromString url |> urlRecombiner |> chompTrailingUrlSlash
+
+
+isValidUrl : String -> Bool
```

Comment:
I know I am a bit late to the party but if this returns `Just Url` we have a valid url if it returns `Nothing` we do not.

https://package.elm-lang.org/packages/elm/url/latest/Url#fromString